src/client/game/game.ts

```diff
@@ -202,18 +202,12 @@ export default class Game {
     // Register restart handler if user is an admin
     const token = bd.meetingToken;
     if (token) {
-      registerRestartButtonListener(() => {
-        try {
-          // gameID is identical to the room name
-          if (!claimsAreValid(token, bd.gameID)) {
```

Comment:
Removing validation as it is no longer used for our non-Daily game host feature (ie restarting the game)